MayssaraOmar/assignment3/Word search/src/Word search.cpp

```diff
@@ -0,0 +1,48 @@
+#include <map>
+#include "word search.h"
+using namespace std;
+
+void get_words_from_grid(vector<vector<char> > &grid, int row, int col,
+		string word, map<pair<int, int>, bool> &vis,
```

Comment:
Optional: I would avoid shortening variable names like "visited" to "vis", because it makes the code harder to read.